mayastor/src/bin/cli/cli.rs

```diff
@@ -83,6 +85,7 @@ async fn main() -> Result<(), Status> {
         ("nexus", Some(args)) => nexus_cli::handler(ctx, args).await?,
         ("pool", Some(args)) => pool_cli::handler(ctx, args).await?,
         ("replica", Some(args)) => replica_cli::handler(ctx, args).await?,
+        ("rebuild", Some(args)) => rebuild_cli::handler(ctx, args).await?,
```

Comment:
I wonder as we can only rebuild a child that is part of a nexus (right?) if we should not put this as a subcommond *under*  the nexus? for example:

```bash
mayastor-client nexus $name rebuild pause
```
... or maybe not... we can always change our mind later (or add them to both)

Reply:
I did start with that but then thought it looked cleaner/more consistent for the command to look like:
`mayastor-client rebuild pause $uuid $uri`

I'd like to leave it like this for the time being, but as you say this can always be changed.